src/common/devtools.js

```diff
@@ -145,6 +150,25 @@ const report =
     )
   ];
 
+const snapshot =
+  (model) => {
+    const id = Date.now().toString(32)
+    const message =
+      [ `\n\n`
+        , `echo '${JSON.stringify(window.application.model.value)}' > `
+        , `./dist/snapshot-${id}.json`
```

Comment:
What about dumping this file in `/tmp/`?

Personnaly, I don't want to pollute `dist`. I will use this features like this:

`servo -w -b --pref dom.mozbrowser.enabled "file:///Users/paul/git/browserhtml/dist/index.html?replay=/tmp/snapshot-1af86t6v7.json"`

Yeah, like I said, I would not add `echo` and `servo` in the output.